test/functional/Java8andUp/src/org/openj9/test/attachAPI/TestJcmd.java

```diff
@@ -122,7 +132,7 @@ public void testCommandNoOptions() throws IOException {
 			String expectedString = commandExpectedOutputs.getOrDefault(command, "Test error: expected output not defined"); //$NON-NLS-1$
```

Comment:
Should the test fail here immediately if the expected output wasn't defined?

Reply:
It is a test coding error if the expected output is not defined, and the test will fail because the expected output is bogus.